components/brave_wallet_ui/common/constants/local-storage-keys.ts

```diff
@@ -9,7 +9,6 @@ export const LOCAL_STORAGE_KEYS = {
   HIDE_PORTFOLIO_NFTS_TAB: 'BRAVE_WALLET_HIDE_PORTFOLIO_NFTS_TAB',
   PORTFOLIO_ASSET_FILTER_OPTION: 'PORTFOLIO_ASSET_FILTER_OPTION',
   PORTFOLIO_TIME_LINE_OPTION: 'PORTFOLIO_TIME_LINE_OPTION',
-  IS_IPFS_BANNER_HIDDEN: 'BRAVE_WALLET_IS_IPFS_BANNER_HIDDEN',
```

Comment:
move to LOCAL_STORAGE_KEYS_DEPRECATED so we can clear the entry from local-storage

Reply:
⬆